docs/pages/reference/graphql-api-reference.md

```diff
@@ -25,39 +25,31 @@ The `Query` type contains the API's top-level entry points for all executable qu
 
 ### `Query.getAvailableAddons`
 
-Query details about Addons available (Eg. Prometheus and Grafana).
+###### **Returns** [`[AddonList!]!`](#addonlist).
```

Comment:
@dhruv0000 for some reason these descriptions disappear, I did check helper.md file again, would be thankful if you could help me out regarding this.

Reply:
How did you generate the docs?
Did you use `make build-graphql-docs`

Reply:
yes I did

Reply:
The descriptions seems to be generated in my setup. This is weird. 

![image](https://user-images.githubusercontent.com/44001539/123491113-94058d00-d633-11eb-90cf-67cc2151bea1.png)

Reply:
Can you check which version of `graphql` and `graphql-docs` you are using, since they are responsible for parsing the `schema.graphql` file.

Reply:
using `graphql 1.12.10` and `graphql-docs 2.0.1` : |